### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by mandyreal**
Default readme updated with project specific details.

### CODE REVIEW

**Comment by pachonk** [comment]
@mandyreal in the main body/description of the PR please put "Resolves #ISSUENUMBER"

File: README.md [hunk 1]
```diff
@@ -1,24 +1,27 @@
...
-* Deployment instructions
-
-* ...
+# Tenacious [![Build Status](https://travis-ci.org/emerald-rubies/tenacious.svg?branch=master)](https://travis-ci.org/emerald-rubies/tenacious)
+
+This is an invetory web app built using Ruby on Rails by a group of developers who belong to the following teams:
+* emeralds - developers new to the language and learning important aspects and best practices of software development process in an OSS environment
+* rubies - devs looking to level up their mentoring skills (who are great mentors to start with)  
+* And last but not the least are the 'sapphires' - devs who know Ruby and probably working as Ruby devs but looking to improve soft skills
+## Project Goals
```
Comment (pachonk) [inline README.md]: Need an additional line break before AND after headings.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,27 @@
...
+The goal of this project is to build an inventory system from scratch by implementing one user story at time.
+## How to set the project up and running
+Pre-requisites : node and yarn.  After cloning the repo, run the following:
+
+```bash
+## to install dependencis, create dev db, run migrations, etc.
+bin/setup
+## to install yarn
+bin/yarn
```
Comment (pachonk) [inline README.md]: This is ran in `bin/setup`

File: README.md [hunk 3]
```diff
@@ -1,24 +1,27 @@
...
-* ...
+# Tenacious [![Build Status](https://travis-ci.org/emerald-rubies/tenacious.svg?branch=master)](https://travis-ci.org/emerald-rubies/tenacious)
+
+This is an invetory web app built using Ruby on Rails by a group of developers who belong to the following teams:
+* emeralds - developers new to the language and learning important aspects and best practices of software development process in an OSS environment
+* rubies - devs looking to level up their mentoring skills (who are great mentors to start with)  
+* And last but not the least are the 'sapphires' - devs who know Ruby and probably working as Ruby devs but looking to improve soft skills
+## Project Goals
+The goal of this project is to build an inventory system from scratch by implementing one user story at time.
+## How to set the project up and running
```
Comment (pachonk) [inline README.md]: Rename to just `Setup`

File: README.md [hunk 4]
```diff
@@ -1,24 +1,27 @@
...
+Pre-requisites : node and yarn.  After cloning the repo, run the following:
```
Comment (pachonk) [inline README.md]: Move above setup and make a bulleted list:
```markdown
## Prerequisites

- Ruby 2.4.1
- NodeJS
- Yarn
```

File: README.md [hunk 5]
```diff
@@ -1,24 +1,27 @@
...
+bin/yarn
+## to set up webpack
+bin/webpack-dev-server
+```
+To run all tests: 
+```ruby
+bundle exec rspec
+```
+## Contributing
+Bug reports and pull requests are welcome on GitHub at https://github.com/emerald-rubies/tenacious/blob/master/CONTRIBUTING.md
```
Comment (pachonk) [inline README.md]:
```markdown
Contribution guide is available [here](https://github.com/emerald-rubies/tenacious/blob/master/CONTRIBUTING.md).
```

File: README.md [hunk 6]
```diff
@@ -1,24 +1,27 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
```
Comment (mandyreal) [inline README.md]: Right. Based on what I did, it involved pretty much the few steps I documented. Even after running bin/setup I had to run yarn install.

File: README.md [hunk 7]
```diff
@@ -1,24 +1,27 @@
...
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
```
Comment (mandyreal) [inline README.md]: Would need help on this, will just put a header for this.

File: README.md [hunk 8]
```diff
@@ -1,24 +1,27 @@
...
-
-* Configuration
-
-* Database creation
```
Comment (mandyreal) [inline README.md]: I understand these are all done when running the bin/setup script and the db need not be created explicitly.

File: README.md [hunk 9]
```diff
@@ -1,24 +1,27 @@
...
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
```
Comment (mandyreal) [inline README.md]: Need help on the details for this one as well. Will put a header for it.

File: README.md [hunk 10]
```diff
@@ -1,24 +1,27 @@
...
-
-* Deployment instructions
```
Comment (mandyreal) [inline README.md]: And this one too. Are we looking at deploying to heroku or others?

File: README.md [hunk 11]
```diff
@@ -1,24 +1,27 @@
...
-
-* ...
+# Tenacious [![Build Status](https://travis-ci.org/emerald-rubies/tenacious.svg?branch=master)](https://travis-ci.org/emerald-rubies/tenacious)
+
+This is an invetory web app built using Ruby on Rails by a group of developers who belong to the following teams:
+* emeralds - developers new to the language and learning important aspects and best practices of software development process in an OSS environment
```
Comment (mandyreal) [inline README.md]: Sure will do.

File: README.md (hunk 2 above)
Comment (mandyreal) [inline README.md]: Based on experience setting up the proj, I had to ran separately though. Will take out then.

**Comment by pachonk** [comment]
@mandyreal ignore the old readme, that's rails. Pay attention to the NEW ISSUE.

File: README.md (hunk 4 above)
Comment (pachonk) [inline README.md]: @mandyreal Actually, rename to System Dependencies

**Comment by pachonk** [comment]
@mandyreal Any progress? I'd like this to be done by our next checkin and if you can't do it, I'm sure somebody else would be willing to snag this issue.

**Comment by mandyreal** [comment]
@pachonk yes please feel free to take over or re-assign to other members. thanks.

**Pull request closed without merging**